testing/simapp/upgrades.go

```diff
@@ -0,0 +1,79 @@
+package simapp
+
+import (
+	storetypes "cosmossdk.io/store/types"
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+
+	consensusparamtypes "github.com/cosmos/cosmos-sdk/x/consensus/types"
+	crisistypes "github.com/cosmos/cosmos-sdk/x/crisis/types"
+
+	capabilitytypes "github.com/cosmos/ibc-go/modules/capability/types"
+	icacontrollertypes "github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts/controller/types"
+	ibcmock "github.com/cosmos/ibc-go/v7/testing/mock"
+	"github.com/cosmos/ibc-go/v7/testing/simapp/upgrades"
+)
+
+// registerUpgradeHandlers registers all supported upgrade handlers
+func (app SimApp) registerUpgradeHandlers() {
```

Comment:
shouldn't this be a pointer receiver?

```suggestion
func (app *SimApp) registerUpgradeHandlers() {
```